pallets/lbp/src/lib.rs

```diff
@@ -627,12 +630,14 @@ pub mod pallet {
 		/// The dispatch origin for this call must be signed by the pool owner.
 		///
 		/// Parameters:
-		/// - `amount_a`: The identifier of the asset and the amount to add.
+		/// - `pool_id`: The identifier of the pool to remove liquidity from
+		/// - `transfer_to_xyk`: Optionally create a XYK pool and transfer there some percentage of the pool liquidity.
+		/// The rest is transferred to LBP owner. If set to zero, XYK pool is not created.
 		///
 		/// Emits 'LiquidityRemoved' when successful.
 		#[pallet::weight(<T as Config>::WeightInfo::remove_liquidity())]
 		#[transactional]
-		pub fn remove_liquidity(origin: OriginFor<T>, pool_id: PoolId<T>) -> DispatchResult {
+		pub fn remove_liquidity(origin: OriginFor<T>, pool_id: PoolId<T>, transfer_to_xyk: Perbill) -> DispatchResult {
```

Comment:
i would consider creating different callabale for this usecase, this doesnt have to be specific just to xyk pool

`transfer_to_xyk` -> `portion_transfered_to_pool`